docs/server/settings.rst

```diff
@@ -250,6 +250,28 @@ Configuration settings for applications used by Pootle.
   - ``label`` specifying the text that will be displayed next to the mark.
 
 
+.. setting:: POOTLE_SCORING_CONF
+
+``POOTLE_SCORING_CONF``
+  Default::
+    {
+        'EDIT_COEF': 5.0/7,
+        'REVIEW_COEF': 2.0/7,
+        'SUGG_COEF': 0.2,
+        'ANALYZE_COEF': 0.1,
+    }
+
+  Parameters:
+
+  - ``EDIT_COEF`` - ``edit cost = NS * EDIT_COEF``
+  - ``REVIEW_COEF`` - ``review cost = NS * REVIEW_COEF``
+  - ``SUGG_COEF`` - ``suggestion cost = NS * SUGG_COEF``
+  - ``ANALYZE_COEF`` - ``reject suggestion cost = NS * ANALYZE_COEF``
```

Comment:
This is about documenting a setting, not how scores are being calculated. So IMHO this change must be done here.